src/internal/schemas/schema.go

```diff
@@ -1,10 +1,53 @@
 package schemas
 
-import "embed"
+import (
+	"bytes"
+	"embed"
+	"fmt"
+	"strings"
+
+	"github.com/santhosh-tekuri/jsonschema/v6"
+)
 
 //go:embed *.json
 var Schemas embed.FS
 
 const (
 	SCHEMA_PREFIX = "oscal_complete_schema-"
+	SCHEMA_PATH   = "https://raw.githubusercontent.com/defenseunicorns/go-oscal/main/src/internal/schemas/"
```

Comment:
Other ideas:
- point to the nist json docs page for the version of the schema used
- use the `$id` this would make it what it currently is (dead link)